src/OneExplorer/OneExplorer.ts

```diff
@@ -81,10 +81,6 @@ enum NodeType {
   derivedModel,
 }
 
-function nodeTypeToStr(t: NodeType): string {
```

Comment:
(7) Replace nodeTypeToStr with node.typeAsString getter

It is moved into Node as a getter